test/task/MonitorTask.test.ts

```diff
@@ -958,6 +958,45 @@ describe('MonitorTask', () => {
       expect(spy.called).to.be.false;
     });
 
+    it('publish event for Signalling Dropped when Web Socket Failed', () => {
```

Comment:
Can you add tests for `meetingReconnected` event as well? Look into `DefaultAudioVideoController.test.ts` for some existing reconnect tests.

Reply:
`DefaultAudioVideoController.test.ts` might be a better candidate for this test. Also, looking at it my request and @simmkyu's request is quite identical. So, I will resolve my comment so you can just focus on Kyu's comment.